tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -15,6 +15,9 @@
 from tendrl.gluster_integration import ini2json
 from tendrl.commons.utils.time_utils import now as tendrl_now
 
+# For 10 min (sync_time(10) + 590)
+SYNC_TTL = 590
```

Comment:
sync_time +15 is not enough for sds sync because it depends on number of bricks and volumes and peers. If number of volumes are very high then some volumes are expired before sync happens. I have disused @shtripat about this so we decided  to set 10 min for sds sync